pkg/products/amqonline/reconciler.go

```diff
@@ -312,15 +312,14 @@ func (r *Reconciler) reconcileStandardAuthenticationService(ctx context.Context,
 			return nil
 		},
 	)
+	if err != nil {
+		return integreatlyv1alpha1.PhaseFailed, fmt.Errorf("could not create postgresql for standard auth service: %w", err)
+	}
 
 	if postgres.Status.Phase != cro1types.PhaseComplete {
```

Comment:
@austincunningham Would appreciate your review here as I think this check was added as part of the amq online backup work.
It looks like this check on the status was added above the err nil check (nil check existed first).
Seems like a simple fix to avoid nil pointer, unless we're missing something

Reply:
@david-martin, err check before checking status I agree with your take on it. Should catch nil pointers .